spec/controllers/teams/users_controller_spec.rb

```diff
@@ -0,0 +1,114 @@
+require 'rails_helper'
+
+describe Teams::UsersController do
+  include Devise::Test::ControllerHelpers
+
+  let(:user) { create(:user) }
+  let(:team) { create(:team) }
+  let(:user_to_delete) { create(:user) }
+
+  before do
+    allow(controller).to receive(:current_user).and_return(user)
+    sign_in user
```

Comment:
Could you also add a couple of tests for a signed-out user? I just want something confirming that we're checking that the user is authenticated before anything else. since it's a common concern, I'm not sure if we have any shared examples or context already written in the specs.